app/lib/helpers/reportStatusHelpers.tsx

```diff
@@ -25,6 +25,40 @@ export const getDaysUntilDue = (reportDueDate: DateTime) => {
   );
 };
 
+export const getWeeksUntilDue = (reportDueDate: DateTime) => {
+  if (!reportDueDate) {
+    return null;
+  }
+  return Math.floor(
+    reportDueDate.diff(
+      // Current date without time information
+      DateTime.now().setZone("America/Vancouver").startOf("day"),
+      "week"
+    ).weeks
+  );
+};
+
+export const getDisplayDueDateString = (
+  daysUntilDue,
+  weeksUntilDue,
+  selectedDate
```

Comment:
Add types so it's clear when to use DateTime vs. string dates vs. any other formats we might try?